tests/endpoints/v1/events_tests.py

```diff
@@ -16,7 +16,7 @@ def test_search_params(self, client):
         client.events.search(id="id", q="query", rank_level=[4,5], rank__gt=85, country=["NZ", "AU"],
              within__radius="2km", within__longitude=-71.0432, within__latitude=42.346,
              label=["label1", "label2"], category="category",
-             place__scope=["place1", "place2"], place__exact=["place3"],
```

Comment:
Are we not supporting the `__` format? If not, should change all the other param tests as well. If we are, should have tests in both forms for all params.